test/integration/suites/sync-authorized-entries/README.md

```diff
@@ -0,0 +1,9 @@
+# Join Token Suite
+
+## Description
+
+This suite verifies that:
```

Comment:
it looks tome than we must recreate an entry with the same reused id right? 
and verify it is failing as expected, or recreated once we removed the old one, 
maybe you can add more scripts verify that?

Reply:
I think that ends up being difficult to handle in the integration test because of timing. I've added some more checks to the unit tests to verify those conditions. Do you think those are enough or should I see about checking those in the integration test?

Reply:
yeah, it is going difficult to test the edge scenario within an IT